test/celeritas/mat/Material.test.cc

```diff
@@ -107,13 +108,30 @@ class MaterialTest : public Test
   protected:
     void SetUp() override
     {
+        real_type const c_light_sq = std::pow(constants::c_light, 2);
+
         MaterialParams::Input inp;
+
+        // Using nuclear masses provided by Geant4 11.0.3 in MeV
+        inp.isotopes = {{AtomicNumber{13},  // Z
+                         AtomicNumber{27},  // A
+                         units::MevMass{25126.5 / c_light_sq},
+                         "Al27"},
+                        {AtomicNumber{13},  // Z
+                         AtomicNumber{28},  // A
+                         units::MevMass{26058.3 / c_light_sq},
+                         "Al28"}};
+
         inp.elements = {
-            {AtomicNumber{1}, units::AmuMass{1.008}, "H"},
-            {AtomicNumber{13}, units::AmuMass{26.9815385}, "Al"},
-            {AtomicNumber{11}, units::AmuMass{22.98976928}, "Na"},
-            {AtomicNumber{53}, units::AmuMass{126.90447}, "I"},
+            {AtomicNumber{1}, units::AmuMass{1.008}, {}, "H"},
```

Comment:
These should have isotopes too. If you're providing any isotopes to MaterialParams you should provide all of them.

Reply:
Any specific reason to have isotopes in all of them other than consistency?
I've added to a single material to test 2 things at once:
1- Empty isotope cases
2- A small array of isotopes, to see if the MaterialParams construction and ElementView/IsotopeView are working as expected.
If both cases are satisfied, then any case (including real scenarios, with all materials having isotopes) should work.

Reply:
The reason is to add the assertion
```
CELER_EXPECT(inp.isotopes_fractions.empty() == this->has_isotope_data());
```
since I think we only want the cases of "all isotope data is available" and "no isotope data is available". You don't want your isotope sampler to work for some elements but not others. The only time we want to support "no isotope data" is if we're using only simple EM physics.